src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Permissions.cs

```diff
@@ -6,13 +7,17 @@ namespace OrchardCore.Apis.GraphQL;
 
 public class Permissions : IPermissionProvider
 {
+
+    [Obsolete("This property will be removed in future release. Instead use 'CommonPermissions.ExecuteGraphQLMutations'.")]
```

Comment:
These are not properties, but fields. And more importantly, a reference to the project/package name would help those consumers who don't have the whole of OC referenced:

```suggestion
    [Obsolete("This field will be removed in a future release. Instead use 'CommonPermissions.ExecuteGraphQLMutations' from OrchardCore.Apis.GraphQL.Abstractions.")]
```
Especially because there are many `CommonPermissions` classes.

Same for all the others.

Reply:
field is used when you are declaring a private property. a property when it is public. + Does it really matter? I ask this because these changes consume time for the PR creator to do. During a review, we should ask yourself these questions so that the PR creator does not have to spent lots of time doing changes that add no value. At the end of the day, we want to merge acceptable PR without burden the creators with lots of changes "when possible".

Reply:
A field and a property are different language features, and this one here is a field. Both can be public and private (or else) too. I'm only pointing this out because it specified "property"; if it were "This will be removed..." or "This member will be removed.." (which are all alternatives BTW) I wouldn't ask for it to include "field".

You can do a search and replace for this and fix it in seconds though, perhaps less than what it takes to debate this :).

Note the more important second part of my comment about the package though.

Reply:
It's not about the debate here. The issues is that we need to think about the value of the comment if it out weight the effort. Many PR end up consuming lots of man hours from (reviewers and creator) due to this type of comment.

I have added the namespaces to the message for developers.

Reply:
If it's not about this instance here, then let's take this conversation offline.